javascript-local-storage/data.js

```diff
@@ -0,0 +1,14 @@
+/* exported todos */
+
+let todos = [];
+
+const previousTodosJSON = localStorage.getItem('javascript-local-storage');
+
+if (previousTodosJSON !== 'null') {
```

Comment:
No quotes around `null`. Now you're checking for a string of `'null'` rather than `null` itself

Reply:
Thank you Shawn! Re-pushed to github and screenshot below.
<img width="470" alt="Screenshot 2023-11-13 at 08 39 37" src="https://github.com/khanhnguyenq/c1023-code-solutions/assets/114459075/07f03244-a0fd-4f2a-8176-1c4fcadd7e28">